llama-index-integrations/embeddings/llama-index-embeddings-cohere/llama_index/embeddings/cohere/base.py

```diff
@@ -103,16 +105,15 @@ def __init__(
             raise ValueError(
```

Comment:
will this part be problematic? since input_type is useless.
should we remove VALID_MODEL_INPUT_TYPES